src/lib/stores/workstreams/types.ts

```diff
@@ -0,0 +1,56 @@
+export enum WorkstreamState {
+    RFA = "rfa", // Request for Applications. Initial state of Workstream
+    ACTIVE = "active",
+    CLOSED = "closed",
+    CANCELLED = "cancelled",
+}
+
+export enum WorkstreamType {
+    GRANT = "grant",
+    ROLE = "role",
+}
+
+export enum Currency {
+    DAI = "dai",
+}
+
+export interface Timestamp {
+    _seconds: number;
+    _nanoseconds: number;
+}
+
+export interface WorkstreamBase {
+    id: string;
+    state: WorkstreamState;
+    creator: string;
+    created_at: Timestamp;
+    payment_rate?: number;
+    payment_currency?: Currency;
+    title: string;
+    desc: string;
+}
```

Comment:
Wouldn't these have "applications: Application" as well?

Reply:
yep, still need to add that 👍